diff --git a/Cargo.toml b/Cargo.toml
index e197c88..65a5ca6 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -3,6 +3,7 @@ name = "bitcoin-exporter"
 description = "Expose bitcoin core metrics in prometheus format"
 version = "0.5.4"
 edition = "2021"
+repository = "https://github.com/eburghar/bitcoin-exporter.git"
 
 [dependencies]
 anyhow = "1.0.51"
@@ -17,11 +18,11 @@ env_logger = "0.9.0"
 log = "0.4.14"
 bitcoin = "0.27.1"
 serde = "1.0.132"
-serde_yaml = "0.8.23"
+serde_yaml = "0.8.24"
 thiserror = "1.0.30"
 
 [patch.crates-io]
-bitcoincore-rpc = { git = "https://github.com/eburghar/rust-bitcoincore-rpc.git", branch = "master" }
-bitcoincore-rpc-json = { git = "https://github.com/eburghar/rust-bitcoincore-rpc.git", branch = "master" }
-# bitcoincore-rpc = { path = "../bitcoincore-rpc/client" }
-# bitcoincore-rpc-json = { path = "../bitcoincore-rpc/json" }
+# bitcoincore-rpc = { git = "https://github.com/eburghar/rust-bitcoincore-rpc.git", branch = "master" }
+# bitcoincore-rpc-json = { git = "https://github.com/eburghar/rust-bitcoincore-rpc.git", branch = "master" }
+bitcoincore-rpc = { path = "../bitcoincore-rpc" }
+bitcoincore-rpc-json = { path = "../json" }
